src/api/Utils.js

```diff
@@ -8,7 +8,7 @@ export const BASE_URL = useLocalData
   : 'https://spruce.dlib.indiana.edu';
 
 // Masterfile ID on the server
-export const masterFileID = 'j3860704z';
+export const masterFileID = 'd791sg30j';
```

Comment:
@adamjarling I ingested a new file and this is the new masterfile ID. Earlier one was deleted when cleaning up the server.